src/Tools/Source/RunTests/RunTests.csproj

```diff
@@ -17,6 +17,8 @@
     <PackageReference Include="Microsoft.Win32.Registry" />
     <PackageReference Include="Newtonsoft.Json" />
     <PackageReference Include="Microsoft.TeamFoundationServer.Client" />
+    <!-- fix of vulnerability in 6.0.0 coming via Microsoft.TeamFoundationServer.Client -->
+    <PackageReference Include="System.Security.Cryptography.Xml" VersionOverride="6.0.1" />
```

Comment:
Can we not put this in Versions.props?

Reply:
If we don't mind using 7.0.1 instead of 6.0.1 (since some other projects use 7.0.1 transitively), we can.